metadata.md

```diff
@@ -28,58 +38,87 @@ There may also be any number of delegated target roles.
+The snapshot.json metadata file lists hashes and sizes of all metadata files
+other than timestamp.json. This file ensures that clients will see a consistent
+view of all files on the repository. That is, metadata files (and thus target
+files) that existed on the repository at different times cannot be combined
+and presented to clients by an attacker.
 
 ​See [example](https://raw.githubusercontent.com/theupdateframework/tuf/develop/tests/repository_data/repository/metadata/snapshot.json) of Snapshot metadata.
 
 ## Timestamp Metadata (timestamp.json)
 
 Signed by: Timestamp role.
 
-The timestamp.json metadata file lists the hashes and size of the snapshot.json file. This is the first and potentially only file that needs to be downloaded when clients poll for the existence of updates. This file is frequently resigned and has a short expiration date, thus allowing clients to quickly detect if they are being prevented from obtaining the most recent metadata. An online key is generally used to automatically resign this file at regular intervals.
-
-There are two primary reasons why the timestamp.json file doesn't contain all of the information that the snapshot.json file does.
-
-* The timestamp.json file is downloaded very frequently and so should be kept as small as possible, especially considering that the snapshot.json file grows in size in proportion to the number of delegated target roles.
-* As the Timestamp role's key is an online key and thus at high risk, separate keys should be used for signing the snapshot.json metadata file so that the Snapshot role's keys can be kept offline and thus more secure.
+The timestamp.json metadata file lists the hashes and size of the snapshot.json file.
+This is the first and potentially only file that needs to be downloaded when
+clients search for updates. It is frequently resigned, and
+has a short expiration date, thus allowing clients to quickly detect if they are
+being prevented from obtaining the most recent metadata. An online key is
+generally used to automatically resign this file at regular intervals.
+
+There are two primary reasons why the timestamp.json file doesn't contain all of
+the information that the snapshot.json file does.
+
+* The timestamp.json file is downloaded very frequently and so should be kept as
+small as possible, especially considering that the snapshot.json file grows
+proportionally with the number of delegated target roles.
+* As the Timestamp role's key is an online key and thus at high risk, separate
+keys should be used for signing the snapshot.json metadata file so that the
+Snapshot role's keys can be kept offline, and thus more secure.
```

Comment:
Adopters often wonder why we don't combine the Timestamp and Snapshot files.  Although we seem to cover it here, we should consider expanding the rationale since user still bring it up.

There is a discussion on our mailing list that covers it.
https://groups.google.com/forum/#!msg/theupdateframework/BoXjULlRNTw/gFtrmVPMAgAJ

Reply:
Maybe this is a change you should do, so we make sure its correct content-wise?

Reply:
@JustinCappos Do you think the Timestamp section of `metadata.md` adequately explains why we separate Timestamp and Snapshot?

It seems to, although I suggest that we explicitly use "combine" when talking about the reasons for why these two roles are not one role.  It instead says, "There are two primary reasons why the timestamp.json file doesn't contain all of the information that the snapshot.json file does," which I think can be missed by some readers.

Reply:
No, I don't like this as much.  Timestamp may be given to mirrors, etc. also.  I do think the size difference matters and is important, but snapshot often does need to be online.  

My feedback is mostly that I feel the emphasis is wrong, not so much the content.

Reply:
Okay, I revised the text here: https://github.com/vladimir-v-diaz/vladimir-v-diaz.github.io/pull/4/commits/2aaddacac1d6944c99e54deff639544efbcad0d4

@jhdalek55 Feel free to revise it, if needed.